cmd/transporter/javascript_builder.go

```diff
@@ -295,14 +301,35 @@ func (js *JavascriptBuilder) Build() error {
 
 // Run runs each of the transporter pipelines sequentially
 func (js *JavascriptBuilder) Run() error {
+	var g group.Group
 	for _, p := range js.pipelines {
-		err := p.Run()
-		if err != nil {
-			return err
-		}
+		g.Add(func() error {
+			return p.Run()
+		}, func(error) {
+			p.Stop()
```

Comment:
as I've been looking through other areas of the code, I think we'll be able to use the group package more to eliminate some of the complexities around starting/listening/stopping so wanted to get it in and get some feedback :)

I'm good with not using it for now as well.